website/docs/setup/guides/advanced-configs.md

```diff
@@ -180,4 +180,5 @@ $ vector -c ./configs/foo.toml ./configs/bar.toml
 If you have a large chain of components it's a good idea to break them out into
 individual files, each with its own unit tests.
 
+
```

Comment:
This newline:

![](https://media.giphy.com/media/GVSFFrwetYEfe/giphy-downsized.gif)